spec/inputs/eventlog_spec.rb

```diff
@@ -1,7 +1,11 @@
 require "logstash/devutils/rspec/spec_helper"
-require 'logstash/inputs/eventlog'
 
-describe LogStash::Inputs::EventLog, :windows => true do
+describe "LogStash::Inputs::EventLog", :windows => true do
+
+  before(:all) do
+    require 'logstash/inputs/eventlog'
```

Comment:
this was working before because we were requiring in the `register` method. moving back the require in to file top has this side effect.